File: docs/assignments/39-elixir-make-it-pass/clamp.exs
defmodule Clamp do
  # Clamp x into the inclusive range [lo, hi].
  def clamp(x, lo, _hi) when x < lo, do: lo
  def clamp(x, _lo, hi) when x > hi, do: x      # <-- one of these lines is wrong
  def clamp(x, _lo, _hi), do: x
end

File: docs/assignments/63-rust-make-it-pass/clamp.rs
// Clamp x into the inclusive range [lo, hi].
pub fn clamp(x: i64, lo: i64, hi: i64) -> i64 {
    if x < lo {
        return lo;
    }
    if x > hi {
        return x; // <-- one of these lines is wrong
    }
    x
}
